tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "jsx": "preserve",
     "allowJs": true,
     "checkJs": false,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
I don't know if I understand correctly the explanation in the link you added in the description, but doesn't it say that for libraries, you should not use `'bundler'`?